### PR TITLE
fix: [CDS-42997]: fixed wrong icon for azure/aws infra at execution studio/diagrams

### PR DESCRIPTION
**Pull request by Sagar-Rajput7**
#### Screenshots

| Before | After |
|:---:|:---:|
| image | image |

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Comment by diptiman-adak** [comment]
Quality gate Quality gate: **_Passed_** 
 


            
 No issues Everything looks good
 [Details](https://app.embold.io/organization/gh/harness/repositories/e864badd0b95ab81c060b15b85f15701/pull-request/11165)

**Comment by Sagar-Rajput7** [comment]
retrigger titlecheck

**Comment by Sagar-Rajput7** [comment]
retrigger titlecheck

File: src/modules/70-pipeline/utils/executionUtils.ts [hunk 1]
```diff
@@ -792,6 +798,7 @@ export function getIconDataBasedOnType(nodeData?: ExecutionNode): {
     if (nodeData.stepType === StepType.ResourceConstraint) {
       return { icon: 'traffic-lights', iconSize: 40 }
     }
+    console.log(nodeData)
```
Comment (vkbansal-harness) [inline src/modules/70-pipeline/utils/executionUtils.ts:801]: remove this console log

Comment (Sagar-Rajput7) [reply src/modules/70-pipeline/utils/executionUtils.ts:801]: Sure, thanks

**Review by sonaligoyal-harness — APPROVED** [review]

**Review by vkbansal-harness — APPROVED** [review]

**Pull request merged**